### PR TITLE
Add a simple single page platform example

### PR DESCRIPTION
**Pull request by johnman**

### CODE REVIEW

File: how-to/use-a-manifest/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## What you can do with this repository
+
+This repository contains examples showing how to configure your application differently based on what you are trying to do.
+
+### Examples
+
+| Example                                             | Description                                                                                                                                                                                                                            |
+| --------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [How To - Create a single page platform](./create-a-single-page-platform)              | This is an example of how start your move to a platform by creating a basic platform manifest that defines a single view to load a site's primary url and hides the tab to make this platform app have a single frame. |
```
[the remark this answers is not in the document]
Comment (johnman) [reply how-to/use-a-manifest/README.md]: word?

File: package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
 		"package-for-github": "node ./scripts/package.js --location github",
 		"package-for-aws": "node ./scripts/package.js --location aws",
 		"prettier": "prettier --config .prettierrc --write .",
-		"eslint": "eslint . --ext .js,.mjs,.ts",
+		"eslint": "eslint . --ext .js,.mjs,.ts --fix",
```
Comment (obany) [inline package.json:11]: Can we have a different command for fix, I have seen fix have unintended consequences

**Review by obany — CHANGES_REQUESTED** [review]

File: how-to/use-a-manifest/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (obany) [reply how-to/use-a-manifest/README.md]: This is an example of how **_to_** start

[the remark this answers is not in the document]
Comment (johnman) [reply how-to/use-a-manifest/README.md]: done

File: package.json (hunk 2 above)
Comment (johnman) [reply package.json:11]: sorry that slipped in.

**Review by obany — APPROVED** [review]

**Pull request merged**